cfme-performance/tests/workloads/test_provisioning.py

```diff
@@ -82,16 +83,19 @@ def cleanup_workload(scenario, from_ts, vms_to_cleanup):
 
     while ((time.time() - starttime) < total_time):
         start_iteration_time = time.time()
-
-        total_provisioned_vms += 1
-        provisioned_vms += 1
-        vm_to_provision = '{}-provision-{}'.format(
-            time.strftime('%Y%m%d%H%M%S'), str(total_provisioned_vms).zfill(3))
-
-        guid_to_provision, provider_name = next(guid_cycle)
-        provider_to_provision = cfme_performance['providers'][provider_name]
-        provision_order.append((vm_to_provision, provider_name))
-        provision_vm(vm_to_provision, guid_to_provision, provider_to_provision['vlan_network'])
+        provision_list = []
+        for i in range(number_of_vms):
+            total_provisioned_vms += 1
+            provisioned_vms += 1
+            vm_to_provision = '{}-provision-{}'.format(
+                time.strftime('%Y%m%d%H%M%S'), str(total_provisioned_vms).zfill(4))
```

Comment:
Is the timestamp changing for every single vm on here?  I think this is a "bug" when we first introduced this, but ideally the timestamp is just whenever we start this provisioning scenario.  That kinda indicates any left over vms will all have the same original timestamp and we can track it down incase we queue up multiple runs of the same test and multiple vms are left over.  That is how I had this in the old framework.